### PR TITLE
use lambda where possible

### PR DESCRIPTION
**Pull request by Jimexist**

### CODE REVIEW

**Comment by bbakerman** [comment]
So in implementation terms I would accept this PR

But the readme examples I dont want to include.

A lambda data fetcher is efficient for some one who truly groks the programming model.

But the README.md examples (which come from that readme.java) are there to be explicit in showing the code

     DataFetcher<Foo> fooDataFetcher = new DataFetcher<Foo>() {
             @Override
             public Foo get(DataFetchingEnvironment environment) {
                 // environment.getSource() is the value of the surrounding
                 // object. In this case described by objectType
                 Foo value = perhapsFromDatabase(); // Perhaps getting from a DB or whatever
                 return value;
             }
 vs

        DataFetcher<Foo> fooDataFetcher = environment -> {
             // environment.getSource() is the value of the surrounding
             // object. In this case described by objectType
             Foo value = perhapsFromDatabase(); // Perhaps getting from a DB or whatever
             return value;
          };

The second case does not tell you the DateFetchingEnvironment etc...

Can you please take out the lambdas from example code please for this PR

File: src/test/groovy/graphql/GarfieldSchema.java [hunk 1]
```diff
@@ -149,17 +149,14 @@ public GraphQLObjectType getType(Object object) {
             .name("Pet")
```
Comment (bbakerman) [inline src/test/groovy/graphql/GarfieldSchema.java]: This is an example linked from the README.  Dont do this one please

File: src/test/groovy/readme/ReadmeExamples.java [hunk 2]
```diff
@@ -116,17 +116,14 @@ void unionType() {
                 .name("Pet")
```
Comment (bbakerman) [inline src/test/groovy/readme/ReadmeExamples.java]: The these out for uber clarity

**Comment by Jimexist** [comment]
@bbakerman good point, reverted `src/test/groovy/graphql/GarfieldSchema.java` and `src/test/groovy/readme/ReadmeExamples.java`

**Pull request merged**